internal/auth/yaml_storage.go

```diff
@@ -93,6 +96,10 @@ func (y *YamlStorage) RemoveCalendarAuth(calendarID string) error {
 
 func (y *YamlStorage) writeFile(cals []CalendarAuth) error {
 	var writer io.Writer
+	if strings.HasPrefix(y.StoragePath, "~/") {
+		usr, _ := user.Current()
```

Comment:
Please use `os.UserHomeDir()` instead (https://pkg.go.dev/os@go1.21.2#UserHomeDir). These functions also require error handling, `usr` is null if the lookup fails.